modules/scrollback/index.js

```diff
@@ -539,6 +544,11 @@ module.exports.getScrollbackForSpecs = function(channel, specs, cb) {
 module.exports.getFormattedScrollbackLinesFromRanges = function(channel, input, cb) {
   var specs = module.exports.parseSpecs(input);
 
+  if(specs.error !== undefined) {
+    this.log.debug(specs.err);
```

Comment:
nix this; since this export is called by other modules, the bot won't have set this for it regardless.